components/brave_wallet_ui/mock-data/apps-list.ts

```diff
@@ -0,0 +1,49 @@
+import { AppsListType } from '../constants/types'
+const compoundIcon = require('../assets/app-icons/compound-icon.png')
+const makerIcon = require('../assets/app-icons/maker-icon.jpeg')
+const aaveIcon = require('../assets/app-icons/aave-icon.jpeg')
+const openSeaIcon = require('../assets/app-icons/opensea-icon.png')
+const raribleIcon = require('../assets/app-icons/rarible-icon.png')
+
+export const AppsList: AppsListType[] = [
+  {
+    category: 'Defi Apps',
+    appList: [
+      {
+        name: 'Compound',
+        description: 'Unlock a universe of open financial applications.',
+        url: 'https://app.compound.finance/',
+        icon: compoundIcon
+      },
+      {
+        name: 'MakerDAO',
+        description: 'Maker - stablecoin, loans and governance...',
```

Comment:
I think all these Descriptions should also be marked as localizable

Reply:
Added! :)